Sources/IssueReporting/ReportIssue.swift

```diff
@@ -73,6 +58,25 @@ public func reportIssue(
         )
       }
     }
+    return
+  }
+
+  switch context {
+  case .swiftTesting:
+    _recordIssue(
+      message: message(),
+      fileID: "\(IssueContext.current?.fileID ?? fileID)",
+      filePath: "\(IssueContext.current?.filePath ?? filePath)",
+      line: Int(IssueContext.current?.line ?? line),
+      column: Int(IssueContext.current?.column ?? column)
+    )
+  case .xcTest:
+    _XCTFail(
+      message().withAppHostWarningIfNeeded() ?? "",
+      file: IssueContext.current?.filePath ?? filePath,
+      line: IssueContext.current?.line ?? line
+    )
+  @unknown default: break
```

Comment:
Do you think it's worth reporting issues from this branch? From a practical perspective I can't see this unknown branch ever hitting, though, so this should be OK.